common/include/pcl/PCLPointCloud2.h

```diff
@@ -4,9 +4,11 @@
    #error USE_ROS setup requires PCL to compile against ROS message headers, which is now deprecated
 #endif
 
+#include <ostream>
 #include <string>
+#include <utility>
```

Comment:
I assume this is for `std::pair`? Can move to the cpp then.

Reply:
Done. Will push soon